src/Analysis/Ast/Test/TypingTests.cs

```diff
@@ -135,7 +135,7 @@ import io
 ";
             var analysis = await GetAnalysisAsync(code);
             analysis.Should().HaveVariable("Foo").OfType("Foo")
-                .And.HaveVariable("foo").OfType("Foo");
+                .And.HaveVariable("foo").OfType("dict");
```

Comment:
Actually `NewType` is a distinct type so it should remain Foo (with base of dict). It acts like `class Foo(dict)`

Reply:
When I run the following script:
```
from typing import NewType

Foo = NewType('Foo', dict)
foo: Foo = Foo({ })

print(type(foo))
```
It prints out `<class 'dict'>` so I think it's safe to change this test.

Reply:
From the point of runtime it is a `dict` like in C `typedef int BOOL` does not actually introduce boolean type. But from type checking perspective it is not a plain `dict` so calling `def func(x: Foo)` with `dict` is wrong as is calling `void func(BOOL x)` in C with `int`. Linters are augmented to check for this.

Reply:
Another thing is what tooltip is going to show. As with `x: List[str]` where it says that `x` is `List[str]` and not `list` (which it actually is at runtime), in `x:Foo` the tooltip should display that `x` is `Foo` and not `dict`.

Reply:
Ok I've changed the test